IngestionGrain/IGrainFactory.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Orleans;
+using System;
+
+namespace CommunAxiom.Commons.Client.Grains.IngestionGrain
+{
+    public interface IGrainFactory
```

Comment:
This should be refactored into its own project in the root of the solution. Project name: Orleans, Base namespace: CommunAxiom.Commons.Orleans